lib/Common/PlatformAgnostic/Arrays.h

```diff
@@ -0,0 +1,35 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+#pragma once
+
+namespace PlatformAgnostic
+{
+namespace Arrays
+{
+    class Utility
```

Comment:
Not sure we need an extra wrapper class around this -- I think `PlatformAgnostic::Arrays::GetLocaleSeparator` is enough here.